src/libs/auth.test.js

```diff
@@ -0,0 +1,144 @@
+import { parseToSDetails } from 'src/libs/auth'
+
+
+const tosDetailsDefault = {
+  currentVersion: '1',
+  isEnabled: true,
+  isGracePeriodEnabled: true,
+  userAcceptedVersion: '1'
+}
+
+const parsedTosDetailsDefault = {
+  isGracePeriodEnabled: true,
+  currentVersion: '1',
+  userAcceptedVersion: '1',
+  userCanUseTerra: true,
+  showTosPopup: false
+}
+
+describe('Terms of Service Detail Parser gives access', () => {
```

Comment:
nit:  describe should refer to a noun, not a verb:
option1: rename these to 'Terms of Service Detail Parser - Access' and 'Terms.... - ToS Popup'
option2: rename to 'Terms of Service Detail Parser' and add nested describe blocks for 'Access' and 'ToS Popup'